### PR TITLE
HJ63P95U: Modify event recorder to use new IAM authentication to DB

### PR DESCRIPTION
**Pull request by chrisclayson**
IAM database authentication is now enabled on RDS and new user, `iam_writer`, has been created.
This change alters the recorder, and import service, to get an IAM authentication token if the
database password is not provided in the environment (in the `ENCRYPTED_DATABASE_PASSWORD` environment variable`).

### CODE REVIEW

File: src/event_handler.py [hunk 1]
```diff
@@ -21,19 +22,25 @@ def store_queued_events(_, __):
...
+        logger.info('Got decryption key from environment variable')
     else:
-      encrypted_decryption_key = fetch_decryption_key()
-      logger.info('Got decryption key from S3')
+        encrypted_decryption_key = fetch_decryption_key()
+        logger.info('Got decryption key from S3')
     decryption_key = decrypt(encrypted_decryption_key)
     logger.info('Decrypted key successfully')
 
+    dsn = os.environ['DB_CONNECTION_STRING']
```
Comment (brenetic) [inline src/event_handler.py]: Deep Space Nine?

**Comment by brenetic** [comment]
Ah, Data Source Name. Google remains my friend. I third the request for an example of what a DSN looks like

File: src/event_handler.py (hunk 1 above)
Comment (brenetic) [reply src/event_handler.py]: Maybe the envar should be called `DATA_SOURCE_NAME`?

Comment (chrisclayson) [reply src/event_handler.py]: Maybe a future change ;-)

**Comment by chrisclayson** [comment]
Updated `README.md` to include section on the environment variables.

File: README.md [hunk 2]
```diff
@@ -65,3 +65,18 @@ in all events being exported from the database. See [Mongo Docs](https://docs.mo
 You should then download this file, over a secure connection, and upload to the S3 bucket configured as the trigger
 for the import_handler Lambda, this typically has the name `govukverify-event-importing-system-<environment>-import-files`
  
+### Environment Variables
+
+The following environment vars are should be defined in the lambda function:
```
Comment (alexandergeegds) [inline README.md]: `are should` 😞

**Pull request merged**